assets/demos/deploy-demoapp.ps1

```diff
@@ -0,0 +1,11 @@
+az group create --name "benchpress-rg-${env:ENVIRONMENT_SUFFIX}" --location "${env:LOCATION}"
```

Comment:
As discussed, can you please check if ENVIRONMENT_SUFFIX is set? If it is not set, then generate a new environment suffix in the script. This way a local developer can easily launch the demo app without needing to set the environment variable. Thanks.

Reply:
👍